runtime/service.go

```diff
@@ -317,6 +321,10 @@ func (s *service) StartShim(shimCtx context.Context, containerID, containerdBina
 	_, err = fcControlClient.CreateVM(shimCtx, &proto.CreateVMRequest{
 		VMID:                  s.vmID,
 		ExitAfterAllTasksGone: exitAfterAllTasksGone,
+		// if containerCount is zero, that signifies that the VM ID has been
+		// specified or CreateVM was not called meaning that the default value will
+		// be used instead of zero, since zero is considered to be "unset"
```

Comment:
I get what containerCount == 0 signifies to the code, but what I don't get, as a reader of the code, is what that actually means in the context of the application. That's what I'd like this comment to address. It's less about what the code is doing and more about why it's doing it.

Reply:
Let's just nuke this comment altogether. It doesn't really contribute anything, and we aren't handling the containerCount==0 special case here anyway, so it doesn't belong here.

Reply:
Done